test/utils/deploy/deploy.go

```diff
@@ -48,10 +48,13 @@ import (
 	e2epause "github.com/openshift-kni/numaresources-operator/test/utils/objects/pause"
 )
 
-type NroDeployment struct {
-	McpObj *machineconfigv1.MachineConfigPool
-	KcObj  *machineconfigv1.KubeletConfig
-	NroObj *nropv1.NUMAResourcesOperator
+type Deployer interface {
+	// Deploy deploys NUMAResourcesOperator object create other dependencies
+	// per the platform that implements it
+	Deploy()
```

Comment:
why does teardown takes a timeout (nice!) and deploy does not?

Reply:
it's just a renaming of the `OverallDeployment` function which didn't take either